### PR TITLE
MIA-FTWD-Rob Gelbman, Crystal Meneses-lab-javascript-clue

### PR DESCRIPTION
**Pull request by RobGelbman**

### CODE REVIEW

File: starter-code/src/clue.js [hunk 1]
```diff
@@ -1,94 +1,84 @@
...
+  return misteryEnvelope
+}
+
+
+function revealMistery (misteryEnvelope){
+
+  //console.log(misteryEnvelope);
+  result = misteryEnvelope[0].firstName + " " + misteryEnvelope[0].lastName + ' killed Mr.Boddy using the ' + misteryEnvelope[1].weapon + " in the " + misteryEnvelope[2].room;
+  return result;
+}
```
Comment (MarcosPalaciosIII) [inline starter-code/src/clue.js]: Your getting an error here on Jasmine but when you put the code on repl.it it works fine. overall you did pass the jasmine test.

**Comment by MarcosPalaciosIII** [comment]
good job on the test. you should have passed but there was an error on test. keep up the good work.

**Comment by RobGelbman** [comment]
The error in Jasmine is because I renamed the variable names in the object. If you want, I can push the fixed version (it passes all the Jasmine tests) I have on my laptop to GitHub.

**Comment by MarcosPalaciosIII** [comment]
Nah its fine I saw that u changed it. Everything functions right. Your good

On Apr 11, 2018 6:09 PM, "RobGelbman" <notifications@github.com> wrote:

The error in Jasmine is because I renamed the variable names in the object.
If you want, I can push the fixed version (it passes all the Jasmine tests)
I have on my laptop to GitHub.

Sent from my iPhone

> On Apr 11, 2018, at 10:34 AM, MarcosPalaciosIII <notifications@github.com>
wrote:
>
> good job on the test. you should have passed but there was an error on
test. keep up the good work.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/ironhack-labs/lab-javascript-clue/pull/365#issuecomment-380611542>,
or mute the thread

**Pull request closed without merging**